WcaOnRails/config/locales/en.yml

```diff
@@ -714,10 +727,14 @@ en:
       competition_was: "Competition was %{n_days} ago."
     #context: in the "nearby competitions" list
     nearby_competitions:
-      comp: "competition"
+      #context: pluralization of "x competition(s)" in the sentence "x competition(s) within n km and m days."
+      competitions:
+        zero: "No competitions"
+        one: "1 competition"
+        other: "%{count} competitions"
       label: "Nearby competitions (within %{days} days and %{kms} km)"
       label_admin: "Last %{days} days"
-      nearby_admin: "%{comps} %{comp} within %{kms} km and %{days} days."
+      nearby_admin: "%{x_competitions} within %{kms} km and %{days} days."
```

Comment:
I had to change these in all the other locales too, to avoid a runtime error.
I did not rename the key because:
- all the translated version group "%{comps} %{comp}" the same way
- When they will update the translation in Jonatan's app, a conflict will pop up and they will just have to make the variable change, without having to translate again the sentence.